win_etw_provider/Cargo.toml

```diff
@@ -28,5 +28,8 @@ uuid="1"
 [features]
 std = []
 default = ["std"]
+uuid =["dep:uuid"]
```

Comment:
Is this the "new" way to do optional dependencies?  I've never seen this syntax before, and I don't see it in the TOML manifest reference.  I'm happy to take the change, this is just the first I've seen of this.

Reply:
yes, it seems to be https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies